xml/System.Web.Configuration/HttpRuntimeSection.xml

```diff
@@ -1053,8 +1053,14 @@
 -   `4.0` The <xref:System.Web.HttpRequest> object internally sets a flag that indicates that request validation should be triggered whenever any HTTP request data is accessed. This guarantees that the request validation is triggered before data such as cookies and URLs are accessed during the request. The request validation settings of the  element (if any) in the configuration file or of the  directive in an individual page are ignored.  
   
 -   `2.0`. Request validation is enabled only for pages, not for all HTTP requests. In addition, the request validation settings of the  element (if any) in the configuration file or of the  directive in an individual page are used to determine which page requests to validate.  
+
+-   `0.0`. Request validation is disabled for the application. `0.0` is only recognized in ASP.NET 4.6 and later.
   
- The value that you assign to this property is not validated to match a specific version of ASP.NET. Any numeric value smaller than `4.0` (for example, `3.7`, `2.9`, or `2.0`) is interpreted as `2.0`. Any number larger than `4.5` is interpreted as `4.5`.  
+ The value that you assign to this property is not validated to match a specific version of ASP.NET. 
+
+ In ASP.NET 4.5 and lower any numeric value smaller than `4.0` (for example, `3.7`, `2.9`, or `2.0`) is interpreted as `2.0`. Any number larger than `4.5` is interpreted as `4.5`.  
+
+ In ASP.NET 4.6 and later any numeric value smaller than `4.0` and greater than  `0.0` (for example, `3.7`, `2.9`, or `1.0`) is interpreted as `2.0`. Any number larger than `4.5` is interpreted as `4.5`
```

Comment:
- comma after 4.6
- "smaller than \`4.0\` --> I think it reads better if reversed ("greater than \`0.0` and less than \`4,0\`..."
- It would be good to use the same phrases to describe relationships of magnitude. Here, "smaller than" is used consistently (though my own preference would be for "less than"), but both "greater than" and "larger than" are used.